# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../suspend/MacBook2.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Works as used to be on the 1,1.

  - Rene Rebe <rene@exactcode.de>

--- suspend/whitelist.c.orig	2000-01-01 00:00:00.000000000 +0100
+++ suspend/whitelist.c	2007-06-12 04:10:21.337706992 +0200
@@ -16,7 +16,12 @@
 	{ "IBM",			"",		"ThinkPad X32",	"", RADEON_OFF|S3_BIOS|S3_MODE },
 	{ "Hewlett Packard",	"",	"HP OmniBook XE3 GF           ","", VBE_POST|VBE_SAVE },
 	/* Alexander Wirt */
-	{ "Apple Computer, Inc.",	"MacBook1,1",		"",	"", VBE_SAVE },
+	{ "Apple Computer, Inc.",	"MacBook1,1",		"",	"", S3_BIOS },
+	/* Rene Rebe */
+	{ "Apple Computer, Inc.",	"MacBook2,1",		"",	"", S3_BIOS },
+	{ "Apple Inc.",			"MacBook2,1",		"",	"", S3_BIOS },
+	{ "Apple Computer, Inc.",	"MacPro2,1",		"",	"", S3_BIOS },
 	/* Julien BLACHE jb@jblache.org */
 	{ "Apple Computer, Inc.",	"MacBookPro2,2",	"",	"", VBE_POST|VBE_MODE },
+	{ "Apple Inc.",			"MacBookPro3,1",	"",	"", },
 	/* Felix Rommel, https://bugzilla.novell.com/show_bug.cgi?id=228572 */
